docs/requirement-sets/outlook/preview-requirement-set/outlook-requirement-set-preview.md

```diff
@@ -91,7 +91,7 @@ Added a new object that allows you to manage the delivery date and time of a mes
 
 Added support for the `OnMessageFromChanged` and `OnAppointmentFromChanged` events in [event-based activation add-ins](/office/dev/add-ins/outlook/autolaunch). To learn more about these events, see [Automatically update your signature when switching between mail accounts (preview)](/office/dev/add-ins/outlook/onmessagefromchanged-onappointmentfromchanged-events).
```

Comment:
depending on the response to Divya in the -pr PR, this section may need to be removed altogether...

Reply:
Per PM guidance, we are keeping the beta channel switching guidance in the article until the feature reaches 100% prod.